docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -31,7 +31,11 @@ paths:
               oneOf:
                 - title: AuthorizationChannelType
                   type: string
+<<<<<<< HEAD
+                  enum: &ref_71
+=======
                   enum: &ref_70
+>>>>>>> upstream/master
```

Comment:
hmm merge conflicts

Reply:
hmm seems I never rebuilt the generated files. done.